[Doc] Add clarification on multiprocessing.cpu_count (#23660)
python/cpython: Doc/library/multiprocessing.rst

   use.  The number of usable CPUs can be obtained with
   ``len(os.sched_getaffinity(0))``

>    When the number of CPUs can not be determined a :exc:`NotImplementedError`

Reviewer: ```suggestion
   When the number of CPUs cannot be determined a :exc:`NotImplementedError`
```

Author: Thanks @Fidget-Spinner
